### ISSUE TITLE
feature:FAQ's section can be added

### ISSUE THREAD

**Issue by TanviGandhotra** [sumn2u/learn-javascript#245]
we can add FAQ's section.giving ans to que like imp of javascript.why we need it today.how this website helps

**Comment by sumn2u** [comment]
That's a great suggestion! Adding an FAQ section could definitely benefit users. I'm not entirely sure where the best placement would be, but perhaps it could go on the landing page?

**Comment by TanviGandhotra** [comment]
yes @sumn2u on landing page i will add this.Plzz assign me this issue under hacktoberfest and gssoc-ext

**Comment by RohitPeswani** [comment]
I can help too in adding faq

**Comment by RohitPeswani** [comment]
where do i need to put my css and js files inside asset folder?

**Comment by sumn2u** [comment]
> where do i need to put my css and js files inside asset folder? [@sumn2u](https://github.com/sumn2u)

The project uses a theme called  [gitbook-plugin-theme-creative](https://github.com/sumn2u/gitbook-plugin-theme-creative) which allows you to add custom styles and scripting logic. I suggest creating a FAQ link that opens a new page when clicked.
